### PR TITLE
Add baseline benchmark for sr25519 verification

### PR DESCRIPTION
**Pull request by emostov**
The weight of sr25519 signature verification is a major factor in the performance of the polkadot parachains runtime `enter` inherent and its creation.

skip cumulus-dependent-check

### CODE REVIEW

File: frame/benchmarking/src/baseline.rs [hunk 1]
```diff
@@ -75,6 +89,26 @@ benchmarks! {
...
+		let msg_and_sigs: Vec<_> = (0..sigs_count).map(|j| {
+			let msg = vec![j, j];
+			(msg.clone(), public.sign(&msg).unwrap())
+		})
+		.collect();
+	}: {
+		msg_and_sigs.iter().for_each(|(msg, sig)| {
+			assert_eq!(
+				sig.verify(&msg[..], &public),
+				true
```
Comment (shawntabrizi) [inline frame/benchmarking/src/baseline.rs:107]: would be good to verify that this is exactly what we want to measure versus a batch signature verification?

**Review by shawntabrizi — APPROVED** [review]

File: frame/benchmarking/src/baseline.rs (hunk 1 above)
Comment (emostov) [reply frame/benchmarking/src/baseline.rs:107]: I am most interested in the speed of the single verification like this https://github.com/paritytech/polkadot/blob/5afceea8b77d27ac6c3eaf500160e9f34807c26b/runtime/parachains/src/disputes.rs#L1221

Comment (shawntabrizi) [reply frame/benchmarking/src/baseline.rs:107]: sounds good then

File: frame/benchmarking/src/baseline.rs [hunk 2]
```diff
@@ -75,6 +89,26 @@ benchmarks! {
...
+			);
```
Comment (bkchr) [inline frame/benchmarking/src/baseline.rs:108]:
```suggestion
			assert!(sig.verify(&msg, &public));
```

**Review by bkchr — APPROVED** [review]

**Review by drahnr — APPROVED** [review]

**Comment by emostov** [comment]
bot merge

**Comment by drahnr** [comment]
bot merge

**Pull request merged**